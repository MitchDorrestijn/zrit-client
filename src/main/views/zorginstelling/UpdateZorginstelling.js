/**
 * React related imports
 */
import React from 'react';

/**
 * Self made components imports
 */
import TextContent from '../../components/TextContent';
import ZorginstellingForm from '../../components/ZorginstellingForm';

/**
 * This component puts the textblock and zorginstellingform (for update) together and renders it
 */
const UpdateZorginstelling = (props) => {
	return (
    <div>
      <TextContent title="Zorginstelling bewerken">
				Op deze pagina kunt u de gegevens van een zorginstelling aanpassen / wijzigen.
				De informatie die op dit moment in de velden staat is de informatie die het systeem nu heeft.
				Om deze te bewerken kunt u deze informatie aanpassen en op 'oplaan' drukken.
				Op de zorginstelling te verwijderen drukt u op de knop 'verwijderen'.
      </TextContent>
      <ZorginstellingForm update={true} id={1} />
    </div>
	);
}

export default UpdateZorginstelling;
