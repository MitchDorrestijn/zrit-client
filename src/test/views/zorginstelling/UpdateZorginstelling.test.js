import React from 'react';
import {shallow} from 'enzyme';
import toJSON from 'enzyme-to-json';
import UpdateZorginstelling from '../../../main/views/zorginstelling/UpdateZorginstelling';

const wrapper = shallow(<UpdateZorginstelling />);

test('<UpdateZorginstelling /> should match snapshot', () => {
  expect(toJSON(wrapper)).toMatchSnapshot();
});

test('<UpdateZorginstelling /> should render one <TextContent> component', () => {
  expect(wrapper.find('TextContent').length).toBe(1);
});

test('<UpdateZorginstelling /> should render one <ZorginstellingForm> component', () => {
  expect(wrapper.find('ZorginstellingForm').length).toBe(1);
});
