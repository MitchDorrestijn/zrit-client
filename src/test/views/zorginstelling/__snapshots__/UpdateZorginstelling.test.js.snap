// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UpdateZorginstelling /> should match snapshot 1`] = `
<div>
  <TextContent
    title="Zorginstelling bewerken"
  >
    Op deze pagina kunt u de gegevens van een zorginstelling aanpassen / wijzigen. De informatie die op dit moment in de velden staat is de informatie die het systeem nu heeft. Om deze te bewerken kunt u deze informatie aanpassen en op 'oplaan' drukken. Op de zorginstelling te verwijderen drukt u op de knop 'verwijderen'.
  </TextContent>
  <ZorginstellingForm
    id={1}
    update={true}
  />
</div>
`;
